docs/types/value-types.rst

```diff
@@ -462,10 +462,11 @@ Underscores are only allowed between two digits and only one consecutive undersc
 There is no additional semantic meaning added to a number literal containing underscores,
 the underscores are ignored.
 
-Number literal expressions retain arbitrary precision until they are converted to a non-literal type (i.e. by
+Number literal expressions retain unlimited precision until they are converted to a non-literal type (i.e. by
 using them together with anything other than a number literal expression (like boolean literals) or by explicit conversion).
 This means that computations do not overflow and divisions do not truncate
-in number literal expressions.
+in number literal expressions. This is achieved by representing fractional rational numbers as quotients of integers, and
```

Comment:
Is it important to document how we do it internally? I'm asking because we need to find this sentence and modify it whenever we do a change.

Reply:
Fair enough, however that is the main point of this whole topic. I.e., that fractional rational numbers are represented as fractional rational numbers (which specifically means as quotients of integers. In other words, if that implementation changed, this whole section and every mention of "unlimited precision" (or "arbitrary precision") in the docs would probably have to be revisited.

Reply:
I also think this is unnecessary. It's a compiler implementation detail not necessary to understand the semantics of literals in Solidity.